pyERGM/metrics.py

```diff
@@ -1077,8 +1079,21 @@ def calc_statistics_for_binomial_tensor_distributed(self, tensor_size, p=0.5, nu
         shutil.rmtree(sample_stats_path)
         return whole_sample_statistics
 
+    def remove_feature_by_idx(self, idx: int):
+        metric_of_feat = self.get_metric_by_feat_idx(idx)
+        is_trimmable = metric_of_feat._get_effective_feature_count() > 1
+        if not is_trimmable:
+            print(f"Removing the metric {str(metric_of_feat)} from the collection")
+            sys.stdout.flush()
+            self._delete_metric(metric=metric_of_feat)
+        else:
```

Comment:
I wonder if there's ever a use-case in which a metric is trimmable, and we delete all it's sub-indices so that we're practically left with no metric. So the metric is still there (`self._delete_metric` was never called), but the `indices_to_ignore` list is full with all indices

Reply:
This is already handled - when a metric with multiple features is left with a single feature, it's categorized as "trimmable", because - 
```
is_trimmable = get_effective_feature_count() > 1
```

and so it will be 1, and will be fully removed